### PR TITLE
Refactor into `cpg_workflows` installable package

### PR DESCRIPTION
**Pull request by vladsavelyev**
Refactor the codebase into a installable/importable package called `cpg_workflows`, so jobs can be imported from e.g. AIP. Doesn't change anything functionally, just moving files around.

If we make it a package, perhaps I could also move the `cpg_utils/workflows` module into this repo as well.

### CODE REVIEW

File: configs/thousand-genomes.toml [hunk 1]
```diff
@@ -6,6 +6,7 @@ scatter_count = 20
 first_stages = ['Combiner']
 last_stages = ['LoadVqsr', 'Frequencies']
 output_version = '0.1'
+skip_qc = true
```
Comment (lgruen) [inline configs/thousand-genomes.toml:9]: Was this intentional?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+from setuptools import find_packages, setup
+
+setup(
+    name='cpg-workflows',
+    # This tag is automatically updated by bumpversion
+    version='0.1.0',
+    description='CPG workflows for Hail batch',
```
Comment (lgruen) [inline setup.py:9]:
```suggestion
    description='CPG workflows for Hail Batch',
```

**Review by lgruen — APPROVED** [review]
Moving the `cpg_utils/workflows` module into this repo as well sounds great to me, if it's not too much work!

File: configs/thousand-genomes.toml (hunk 1 above)
Comment (vladsavelyev) [reply configs/thousand-genomes.toml]: fixed

File: setup.py (hunk 2 above)
Comment (vladsavelyev) [reply setup.py]: fixed

**Review by MattWellie — APPROVED** [review]
exciting!

**Pull request merged**